{"name": "test_mul_006", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_file2.Test002 object at 0x000001B42BCAF5C0>\n\n    @pytest.mark.group3\n    def test_mul_006(self):\n        a = 2\n        b = 2\n        mul = a * b\n        print(\"Mul of a * b =\" + str(mul))\n        if mul == 5:\n            assert True\n        else:\n>           assert False\nE           assert False\n\ntest_file2.py:25: AssertionError"}, "attachments": [{"name": "stdout", "source": "7a897bed-4068-4a8b-b4c4-5ccb62564547-attachment.txt", "type": "text/plain"}], "start": 1726213205136, "stop": 1726213205137, "uuid": "aaacfcf5-e559-4272-a95c-eabf661ba869", "historyId": "cdb5b3e79acb6b5df0c56db44f1f660f", "testCaseId": "cdb5b3e79acb6b5df0c56db44f1f660f", "fullName": "test_file2.Test002#test_mul_006", "labels": [{"name": "tag", "value": "group3"}, {"name": "suite", "value": "test_file2"}, {"name": "subSuite", "value": "Test002"}, {"name": "host", "value": "DESKTOP-PD08TMV"}, {"name": "thread", "value": "6060-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_file2"}]}